src/middlewared/middlewared/plugins/smb.py

```diff
@@ -1398,6 +1402,8 @@ async def pool_post_import(middleware, pool):
             ('path', '^', f'{path}/'),
         ])
     ]):
+        await middleware.call('sharing.smb.disable_acl_if_trivial')
+        await middleware.call('sharing.smb.set_timemachine_quotas')
```

Comment:
Have we validated whether this is a correct / supported way of doing time machine quotas over SMB?

Reply:
Yes, reading this plist file is protocol agnostic

Reply:
I'm still not fond of the plist. Is this actually documented by Apple as a method of managing time machine size? I'm asking because there is a documented and supported way of setting quotas on time machine backups in samba:

```
fruit:time machine max size = SIZE [K|M|G|T|P]
```